### ISSUE TITLE
Troubleshooting: The project I specified when running the download script didn't download 

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/OpenScPCA-analysis#161]
If someone using `download-data.py` specifies and project or sample that doesn't exist, it will not fail or warn, just not download samples. We should explain this in the troubleshooting or FAQs and provide instructions about how to list things in the data release bucket to avoid running into the issue.

**Comment by sjspielman** [comment]
> provide instructions about how to list things in the data release bucket to avoid running into the issue.

Quick question here for @jashapiro @jaclyn-taroni - currently I see the release bucket has "folders" of dates. Do we have any plan to maintain a "current" or "release" folder too? I was looking at [these docs](https://github.com/AlexsLemonade/OpenScPCA-admin/blob/main/technical-docs/data-structure-and-releases.md#directory-structure) and didn't see any indication of that, but maybe there are new thoughts on the matter.

**Comment by jaclyn-taroni** [comment]
The most recent dated release is symlinked to `data/current`, see: https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/getting-started/accessing-resources/getting-access-to-data.md#using-the-download-data-script

**Comment by jashapiro** [comment]
> > provide instructions about how to list things in the data release bucket to avoid running into the issue.
> 
> Quick question here for @jashapiro @jaclyn-taroni - currently I see the release bucket has "folders" of dates. Do we have any plan to maintain a "current" or "release" folder too? I was looking at [these docs](https://github.com/AlexsLemonade/OpenScPCA-admin/blob/main/technical-docs/data-structure-and-releases.md#directory-structure) and didn't see any indication of that, but maybe there are new thoughts on the matter.

The only way to have a `current` folder in the bucket would be to duplicate all the data. So no, there are no plans to have a `current` folder in the bucket.

**Comment by sjspielman** [comment]
> The most recent dated release is symlinked to data/current

Yes, this is true locally but was thinking about S3. Since I guess there's no way to symlink that, then nope! I'll explain to go by the most recent date.

**Comment by sjspielman** [comment]
Noting that I guess in the future we might consider adding an option to the download script to list projects and/or libraries in a current release?

**Comment by allyhawkins** [comment]
Not sure how this got reopened, whoops.
